resources/kernel-cache-drop-daemonset.yaml

```diff
@@ -18,6 +18,8 @@ spec:
       #tolerations:
       #- key: node-role.kubernetes.io/master
       #  effect: NoSchedule
+      nodeSelector:
```

Comment:
what about the spec:selector:matchLabels: component?  Does that have no effect?   Perhaps I misunderstood what that does.   [See docs](https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/).  As long as you have tested it and it works, I have no problem with using the node selector.